### ISSUE TITLE
Unable to input scaling factor of 0.01

### ISSUE THREAD

**Issue by fire** [WonderMediaProductions/Maya2glTF#82]
Maya models in the cm scale, but Godot uses m scale, so we want to use scale the entire scene by 0.01, but it is not inputtable.

**Comment by fire** [comment]
![image](https://user-images.githubusercontent.com/32321/65532806-5b121480-deb1-11e9-9f93-d9ff01b9c979.png)

0.01 becomes 0.0

**Comment by fire** [comment]
![image](https://user-images.githubusercontent.com/32321/65532863-74b35c00-deb1-11e9-988f-2a12e1e1b5b8.png)

0.01 becomes 0.0

**Comment by Ziriax** [comment]
I guess you just need to add the `-precision 6` flag to the `floatFieldGrp` in the `maya2glTF-UI.mel` script (not where it is queried, but where the field is created)

Could you give that a try?

**Comment by fire** [comment]
This works.

```mel
global proc maya2glTF_UI()
{
    // This is defined on maya startup
    global string $gMainProgressBar;  

    // This is auto-updated by msbuild 
    string $maya2glTF_version = "V0.9.10 1f6a8bb";

    if (`window -exists maya2glTF_exporter_window`)
        deleteUI maya2glTF_exporter_window;

    string $year = `date -format "YYYY"`;
    string $chars[] = `ls -type character`;

    // Load previous or default values
    string $exportDir = fromNativePath(_getStr("outputFolder", _getDefaultOutputFolder()));
    string $sceneName = _getStr("sceneName", _getDefaultSceneName());
    string $copyright = _getStr("copyright", "(C)"+$year+" "+getenv("username"));
    float $scale = _getFlt("scale", 1);
    string $flags = _getStr("flags", "");
    string $char = _getStr("char", $chars[0]);
    int $anim = _getInt("anim", 1);
    string $clip = _getStr("clip", "clip");
    int $selOnly = _getInt("selOnly", 1);
    int $visOnly = _getInt("visOnly", 1);

    window -title ("maya2glTF") maya2glTF_exporter_window;

    string $header = "Copyright (C)" + $year + " wondermedia.tv         version " + $maya2glTF_version;

    string $frame = `frameLayout -mw 30 -mh 30 ($header)`;

    columnLayout -adjustableColumn true -rowSpacing 10; 

    button -label "assign PBR shader to selection" -command "maya2glTF_assignPbrShader();";
    button -label "ignore selected mesh deformers" -command "maya2glTF_ignoreSelectedDeformers();";

    separator;

    button -label "select all polygon meshes" -command "maya2glTF_polySelectAndFrame();" -bgc (60/255.0) (80/255.0) (120/255.0);

    separator;

    string $folderFLD = `textFieldGrp -fileName $exportDir -label "output folder:" -adjustableColumn 2`;
    string $copyrightFLD = `textFieldGrp -text $copyright -label "copyright:" -adjustableColumn 2`;
    string $sceneNameFLD = `textFieldGrp -text $sceneName -label "scene name:" -adjustableColumn 2`;
    string $scaleFLD = `floatFieldGrp -precision 6 -value1 $scale -label "scale factor:"`;
```

**Comment by Ziriax** [comment]
Fixed by #87
